Sources/BraveWallet/Crypto/NFTImageView.swift

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import SwiftUI
+import BraveUI
+import SDWebImageSwiftUI
+
+struct NFTImageView<Fallback: View>: View {
+  let urlString: String
+  
+  private var fallback: () -> Fallback
+  
+  init(
+    urlString: String,
+    @ViewBuilder fallback: @escaping () -> Fallback
+  ) {
+    self.urlString = urlString
+    self.fallback = fallback
+  }
+  
+  var body: some View {
```

Comment:
I think this view needs `.allowsHitTesting(false)` modifier, gifs are intercepting touches on cells. Commenting here because I think likely just wrap this body in `Group`.

Reply:
fixed in [00c5f80](https://github.com/brave/brave-ios/pull/6534/commits/00c5f80308e952e4dc6914ae61d3bbcddda7d726)
i apply this modifier to `NFTImageView` to avoid using `Group` here